pkg/apis/core/v1alpha1/types_overridepolicy.go

```diff
@@ -84,16 +84,70 @@ type TargetClusters struct {
 // Overriders contains a list of override patches.
 // The order in which the override patches take effect is:
 // - Image
+// - Command
+// - Args
+// - Annotations
+// - Labels
 // - JsonPatch
 type Overriders struct {
 	// Image specifies the overriders that applies to the image.
 	// +optional
 	Image []ImageOverrider `json:"image,omitempty"`
+
+	// Command specifies overriders that applies to the container commands.
+	// +optional
+	Command []EntrypointOverrider `json:"command,omitempty"`
+
+	// Args specifies overriders that applies to the container arguments.
+	// +optional
+	Args []EntrypointOverrider `json:"args,omitempty"`
+
+	// Annotation specifies overriders that applies to the resource annotations.
+	// +optional
+	Annotations []StringMapOverrider `json:"annotations,omitempty"`
+
+	// Label specifies overriders that applies to the resource labels.
+	// +optional
+	Labels []StringMapOverrider `json:"labels,omitempty"`
+
 	// JsonPatch specifies overriders in a syntax similar to RFC6902 JSON Patch.
 	// +optional
 	JsonPatch []JsonPatchOverrider `json:"jsonpatch,omitempty"`
 }
 
+type EntrypointOverrider struct {
+	// ContainerName targets the specified container or init container in the pod template.
+	ContainerName string `json:"containerName"`
+
+	// Operator specifies the operation.
+	// If omitted, defaults to "overwrite".
+	// +kubebuilder:validation:Enum=append;overwrite;delete
+	// +optional
+	Operator string `json:"operator,omitempty"`
+
+	// Value is the value(s) that will be applied to command/args of ContainerName.
+	// Items in Value(empty is not allowed) which will be appended after command/args when Operator is 'append'.
```

Comment:
```suggestion
	// If O  perator is 'append', items in Value (empty is not allowed) will be appended to command/args .
```

Reply:
The code has been updated.

Reply:
Pls fix it for `overwrite` and `delete`. The original sentence structure is a bit weird.

Reply:
The code has been updated.